core/src/main/java/io/seata/core/protocol/RegisterTMRequest.java

```diff
@@ -87,11 +87,4 @@ public short getTypeCode() {
         return MessageType.TYPE_REG_CLT;
     }
 
-    @Override
-    public String toString() {
-        return "RegisterTMRequest{" +
-            "applicationId='" + applicationId + '\'' +
-            ", transactionServiceGroup='" + transactionServiceGroup + '\'' +
-            '}';
-    }
```

Comment:
Why remove this override?

Reply:
`RegisterTMRequest` has no additional filed and implements `toString()` in its parent class `AbstractIdentifyRequest`.